content/docs/studio/user-guide/model-registry/assign-stage.md

```diff
@@ -1,18 +1,29 @@
-# Assign stages to models
+# Assign stage to model version
 
-To manage model lifecycle, you can assign stages (such as `development`,
-`staging`, `production`, etc.) to specific model versions.
+To manage model lifecycle, you can assign stages (such as `dev`, `staging`,
+`prod`, etc.) to specific model versions.
 
 To assign a stage to a model version, Iterative Studio uses [GTO] to create an
 annotated [Git tag][git tag] with the specified stage and version number. Refer
 to the [GTO docs][gto-format] to see the exact format of the Git tag.
 
-You can write CI/CD actions that can actually deploy the models to the different
-deployment environments upon the creation of a new Git tag for stage assignment.
-For that, you can leverage any ML model deployment tool, such as MLEM.
+You can [write CI/CD actions][CI/CD] that can actually deploy the models to the
+different deployment environments upon the creation of a new Git tag for stage
+assignment. For that, you can leverage any ML model deployment tool, such as
+[MLEM].
 
-To assign stages using Iterative Studio, watch this tutorial video or read on
-below:
+You can assign a stage in any of the following ways:
+
+1. Use GTO CLI or API. An example would be
+   [`gto assign s3:mymodel --version v0.0.1 --stage dev`][assign]. In this case,
+   `s3` is a path within a repo to the directory where `dvc.yaml` with model
```

Comment:
Why `s3`? I first thought it was an aws s3 path. Can we use a different path which looks more like a path in the repo?

Reply:
ok, I see that it's how the example monorepo uses it. But still, may be this can be simplified here somehow?

Reply:
Unfortunately, I don't have a better repo at hand. AFAIK, we don't have a proper monorepo to show except for this and https://github.com/iterative/monorepo-test, which is even more strange since we don't claim to support that AFAIK (nesting DVC projects).

Reply:
Oh, forgot about this one https://github.com/aguschin/example-gto-dvc-monorepo. It can be a better fit, although then I need to move it to `iterative` to make it "frozen". It's not auto-generated though.

Reply:
ok. Maybe we can proceed with the way it is now, and when possible update it (after you stabilize the other example repo).

Reply:
@tapadipti, I adapted another repo to demo this. Can you please check out new text to confirm whether it's more clear now? Thanks!

Reply:
Sorry, updated it once again @tapadipti. Should be easier to understand now!

Reply:
Lgtm @aguschin. Thanks for addressing this. I've made one additional tiny suggestion.